Makefile

```diff
@@ -41,6 +41,8 @@ VS_ADDER = add_vs_cmap.py # from nototools
 EMOJI_SRC_DIR ?= png/128
 FLAGS_SRC_DIR := third_party/region-flags/png
 
+CHECK_COVERAGE_PY = check_emoji_sequences.py
```

Comment:
why the variable is called differently from the name of the script? Maybe it's just me but when I read COVERAGE_PY I immediately think of `coverage.py` (https://coverage.readthedocs.io/), the test coverage tool for python

Reply:
+1, coverage means something specific in this context that isn't what we mean here

LINTER_PY or SEQUENCE_CHECK_PY or something like that perhaps?

Reply:
Good one, named it SEQUENCE_CHECK_PY now